src/components/ForgotPassword/ForgotPassword.styles.js

```diff
@@ -0,0 +1,23 @@
+import styled from '@emotion/styled';
+
+import Form from 'components/Form';
+
+export const Input = styled(Form.Input)`
```

Comment:
I think that the approach we are using is to do some `css` /**/ ``  styles and pass them as the "className" prop to override the styles of the base components. @emmaielle thoughts?

Reply:
You are right, that one is another way to solve the same problem, which one should we keep?

Reply:
I think we decided to go with the first approach so that we avoid importing components in the styles files, and also because this is a child component from Form and we thought it didn't look great to do this (refer to https://teamloopstudio.slack.com/archives/CKCG8DL0L/p1592262693006700). 
I'm still unsure about which is better. Your option @Rminetti is simpler to implement though. As long as they are all consistent to one way I like both.

Reply:
I personally like this approach better, but I did not realized that this issue had already been discussed, I'll change it to use the css function